docs/site/TEMP-TEXT-FILES/errors-and-warnings.txt

```diff
@@ -3,26 +3,27 @@ Handling Errors and Warnings
 Errors
 Errors in the survey tool MUST be fixed for the Winning column. Following are common errors that you may encounter and tips on ways to fixing them.
 Can't have same translation as… (Must fix)
-Exactly the same name can't be used for two different items of the same type. 
+Exactly the same name can't be used for two different items of the same type.
 For example, it is an error if "Malawi-Kwacha" is used as the translation for two different currency codes:
 MZN (Mozambican metical), and
 MWK (Malawian Kwachas)
 In the above case, it is a pretty clear mistake. It is not as simple in other cases. Some region names are very similar, for example, and it can be a bit tricky to know how to handle them. Currencies have the problem that a new currency code may have the same name as an obsolete one.
 You can find some guidance under "Unique Names" in the following pages:
-Country/Region Names, 
-City Names, 
+Country/Region Names,
+City Names,
 Currency Symbols & Names
 The characters ‎[…]‎ should not be used (Must fix)
 For what to do, see Characters, in the section Handling Warnings. While these are categorized as warnings, every effort should be made to fix them.
 Unquoted special character '.' in pattern (Must fix)
 Number patterns can only contain an unquoted . when it is the decimal separator.
-In other cases, such as to finish an abbreviation, it must be quoted, as in:    000 tūkst'.'
+In other cases, such as to finish an abbreviation, it must be quoted, as in: 000 tūkst'.'
 For more information, see Number Patterns.
 Errors in Others column can be ignored. For example, this is an example of an Inheritance value in the Others column where the Language (Country/region) names have a collision issue. Do not vote on values that have errors.
+alt-text
```

Comment:
The introduction of "alt-text" looks like an error.

Reply:
alt-text is just what shows up in the text file for any images when I copy directly from the output of markdown. I try to clean them up for the most part but they aren't in the actual md files.

Reply:
got it